src/main/java/com/spikes2212/control/JoinedPID.java

```diff
@@ -0,0 +1,43 @@
+package com.spikes2212.control;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class JoinedPID implements PIDLoop {
```

Comment:
it is better to name this class `JoinedPIDLoop` or something. just so we stick to the naming convention we created for all the other implementations of `PIDLoop`.